### PR TITLE
LRO API annotated with ReturnType.LONG_RUNNING_OPERATION

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**

### CODE REVIEW

**Review by haolingdong-msft — APPROVED** [review]

File: fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentClientMethodTemplate.java [hunk 1]
```diff
@@ -242,7 +242,7 @@ protected void generateLongRunningBeginAsync(ClientMethod clientMethod, JavaType
         boolean mergeContextParameter = settings.getAddContextParameter() && (settings.isContextClientMethodParameter() && contextInParameters(clientMethod));
         String contextParam = mergeContextParameter ? "context" : "Context.NONE";
 
-        typeBlock.annotation("ServiceMethod(returns = ReturnType.SINGLE)");
+        typeBlock.annotation("ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)");
```
Comment (haolingdong-msft) [inline fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentClientMethodTemplate.java:245]: do we need to update LRO related  tests?

Comment (haolingdong-msft) [inline fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentClientMethodTemplate.java:245]: do we need to update LRO related  tests?

Comment (weidongxu-microsoft) [reply fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentClientMethodTemplate.java:245]: No, it is not used by customer. Either it is only informative, or used by azure-core reflex.

**Comment by weidongxu-microsoft** [comment]
test run passed the previous failed ones
https://dev.azure.com/azure-sdk/public/_build/results?buildId=1119146&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ba3cf43a-b2bf-50f9-f95f-93049ca44ad1

**Pull request merged**